src/main/java/picard/sam/AbstractAlignmentMerger.java

```diff
@@ -694,8 +694,7 @@ private void transferAlignmentInfoToFragment(final SAMRecord unaligned, final SA
 
             // if there already is a comment, add second comment with a | separator:
             Optional<String> optionalComment = Optional.ofNullable(unaligned.getStringAttribute(SAMTag.CO.name()));
-            unaligned.setAttribute(optionalComment.map(s -> s + " | ").orElse("") +
-                    SAMTag.CO.name(), "Cross-species contamination");
+            unaligned.setAttribute( SAMTag.CO.name(), optionalComment.map(s -> s + " | ").orElse("")  + "Cross-species contamination");
```

Comment:
The fix looks good to me. It's a shame the API isn't `setAttribute(SAMTag, Object)`.
```suggestion
            unaligned.setAttribute(SAMTag.CO.name(), optionalComment.map(s -> s + " | ").orElse("")  + "Cross-species contamination");
```

Reply:
There's a pr open to do what you suggested.